relay-server/src/endpoints/project_configs.rs

```diff
@@ -27,12 +60,16 @@ fn get_project_configs(
                 let project_state = project_state.ok()?;
                 // If public key is known (even if rate-limited, which is Some(false)), it has
                 // access to the project config
-                if project_state.config.trusted_relays.contains(&public_key) {
+                if project_state
```

Comment:
This also needs to check if the Relay is `internal`, which implicitly gives it access.